### PR TITLE
[Merged by Bors] - chore: Rename intervals to `uIcc`/`uIoc`

### PR DESCRIPTION
**Pull request by YaelDillies**
Match https://github.com/leanprover-community/mathlib/pull/18104

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Set/Intervals/OrdConnectedComponent.lean [hunk 1]
```diff
@@ -85,13 +85,13 @@ theorem ordConnectedComponent_inter (s t : Set α) (x : α) :
 
 theorem mem_ordConnectedComponent_comm :
     y ∈ ordConnectedComponent s x ↔ x ∈ ordConnectedComponent s y := by
-  rw [mem_ordConnectedComponent, mem_ordConnectedComponent, interval_swap]
+  rw [mem_ordConnectedComponent, mem_ordConnectedComponent, uIcc_swap]
```
Comment (kbuzzard) [inline Mathlib/Data/Set/Intervals/OrdConnectedComponent.lean:88]: This `uIcc_swap` seems to be `uIcc_comm` in mathlib3. What is going on here?

**Review by kbuzzard — COMMENTED** [review]
Modulo the swap/comm issue this looks good to me; I am a bit nervous about checking these matching PRs by hand though.

File: Mathlib/Data/Set/Intervals/OrdConnectedComponent.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Data/Set/Intervals/OrdConnectedComponent.lean]: Whoops sorry, my bad. It should be `uIcc_comm`.

**Comment by eric-wieser** [comment]
@YaelDillies, can you fix the conflicts?

**Comment by eric-wieser** [comment]
bors merge

In the absense of any automated tools to help review this, I'm going to assume this patch is ok.
We'll want automated tools eventually anyway to check that we didn't make mistakes during regular porting.

**Pull request closed without merging**